mantle-build/src/rpc.rs

```diff
@@ -3,14 +3,15 @@
 use std::collections::BTreeSet;
 
 use rustc::{
-    hir::{self, def_id::DefId, FnDecl},
+    hir::{self, def_id::DefId, Body, FnDecl},
     ty::{self, AdtDef, TyCtxt, TyS},
     util::nodemap::FxHashMap,
 };
 use syntax_pos::symbol::Symbol;
 
 use mantle_rpc::{
-    Field, Function, Import, Interface, StateConstructor, StateMutability, Type, TypeDef,
+    Field, Function, Import, IndexedField, Interface, StateConstructor, StateMutability, Type,
```

Comment:
is this the time to put each one on a newline?

Reply:
idk, ask rustfmt

Reply:
(read: it's not my choice. this is what rustfmt said it should be)